packages/utils/src/validators.ts

```diff
@@ -0,0 +1,13 @@
+const URL_REGEX =
+    /^(http|ws)s?:\/\/[a-z0-9]([a-z0-9.-]+)?(:[0-9]{1,5})?((\/)?(([a-z0-9-_])+(\/)?)+)$/i;
+
+export const UPPERCASE_RE = new RegExp('^(.*[A-Z].*)$');
```

Comment:
The name is not good:
1. `RE` on the end should be converted to `REGEXP`, we should use shortcuts everywhere or nowhere, so we should do it same way as `URL_REGEX`
2. Name gives you very vague idea what it should do? Is first letter upper case? Is whole text in upper case? Is there any uppercase letter in text? I would recommend rename to smthing like `HAS_UPPERCASE_LATER_REGEXP` it's quite long but in the other it's super explicit

Reply:
or you can move this regexp directly to `hasUppercaseLetter` function, then you don't have to care of naming of it

Reply:
I renamed it to `HAS_UPPERCASE_LATER_REGEXP`.  :+1: